---
permalink: /floriancv/
title: "Curriculum vitae"
author_profile: true
author: "Florian Heigwer"
toc: true
toc_label: "Content"
toc_icon: "fas fa-table"
toc_sticky: true
---

_a more personal description can be found [here](/about/)_

<div style="background-color:rgba(40, 40, 40, 0.0470588); text-align:center; vertical-align: middle; padding:40px 0;">

In my research I aim to understand gene function in health and disease. To this end, I work on the crossroads of high-throughput molecular biology and computational biology to systematically map genes to their function in different biological contexts.  

</div>
<br />
**Key words:** Bioinformatics, Genome Research, Functional Genomics, Image Analysis, Machine Learning

## **Overview**

| :------------- | :----------: | -----------: |
|  **Name** | Florian Heigwer   | **Publications**     | 20     |
| **Nationality** | German |  **h-index** | **15**     |
| **E-mail** | f.heigwer[at]dkfz.de | **ORCID** | [0000-0002-8230-1485](https://orcid.org/0000-0002-8230-1485) |
| **Date of Birth** | 01/29/1988 | **Google Scholar** | [GScholar](https://scholar.google.de/citations?user=QEUKuFMAAAAJ&hl=en) |

## **Education**

| Timing | Degree | Topic |  Grade |
| :-------------  | :----------: | -----------: |
|  Since 02/2018  | Post-Doc   | Functional analysis of genetic interactions using multivariate high-content screening methods    | |
| 10/2013-02/2018 | PhD (Dr. rer. nat.) | Advanced methods for functional genomics screening | _summa cum laude_ |
| 09/2011-09/2013 | M.Sc. Molecular Biotechnology | Bioinformatics Tools for Genome Engineering | 1.1 |
| 10/2008-08/2011 | B.Sc. Molecular Biotechnology | Optimization of Kinetic Microarray-ELISA | 1.6 |
| 1998-22/05/2006 | A-Level degree, Ev. Ratsgymnasium |  | 1.4 |

## **Professional Career**

| Timing | Group | Institute |
| :------------- | :----------: | -----------: |
|  **Since 01/2013** | M. Boutros \([Div. Signaling and Functional Genomics](https://www.dkfz.de/en/signaling/index.php)\) | German Cancer Research Center (DKFZ), Heidelberg, Germany |
| **09/2012-12/2012**   | A. Andersson \([Div. Metagenomics](https://www.scilifelab.se/researchers/anders-andersson/)) |  Science for Life Laboratory, Royal Institute of Technology, Stockholm, Sweden |
| **01/2012-09/2012**   | S.W. Hell ([Div. Nanoscopy, Super Resolution Microscopy](https://www.mr.mpg.de/departments/optical-nanoscopy)) | German Cancer Research Center (DKFZ), Heidelberg, Germany |
| **10/2009-06/2012**   | S. Wölfl ([Div. Bioanalytics](https://www.ipmb.uni-heidelberg.de/biologie/woelfl/index.html)) | Institute for Pharmacy and Molecular Biotechnology (IPMB), University of Heidelberg, Heidelberg, Germany | |

## **Honors**

| :------------- | :----------: |
|  **09/2012-12/2012** | ERASMUS Placement Scholarship |
| **01/2017**   | HBIGS-Travel grant |
| **09/2018**   | CytoData-2018 Travel grant |
| **09/2018**   | CyoData-2018 Challenge Winning Team (10.000$ Nvidia Card) |

## **Publications**

| authors | title | journal |year| type|
| :------------- | :----------: | ---------: | -----: | -----: |
J. Neufeldt C., Cerikan B., Cortese M., Frankish J., Lee J., Plociennikowska A., **Heigwer F.** et al.| SARS-CoV-2 infection induces a pro-inflammatory cytokine response through cGAS-STING and NF-κB| BioRxiv |2020|preprint|
**Heigwer F.**, Boutros M.| Cloud-Based Design of Short Guide RNA sgRNA Libraries for CRISPR Experiments| Methods in molecular Biology |2020|research article|
Port F., Strein C., Stricker M., Rauscher B., **Heigwer F.** et al.|A large-scale resource for tissue-specific CRISPR mutagenesis in Drosophila| eLife |2020|research article|
Rauscher B., Henkel L., **Heigwer F.**, Boutros M.| Lineage specific core-regulatory circuits determine gene essentiality in cancer cells| BioRxiv |2019|preprint|
**Heigwer F.**, Scheeder C. et al.| Time-resolved mapping of genetic interactions to model rewiring of signaling pathways| eLife |2018|research article|
Scheeder C., **Heigwer F.**, Boutros M.| Machine learning and image-based profiling in drug discover| Curr. Opinions in Systems Biology |2018| Review|
**Heigwer F.**, Port F., Boutros M.| RNA Interference Screening in Drosophila| Genetics |2018| Review|
Rauscher B., **Heigwer F.**, Henkel L. et al.| Toward an integrated map of genetic interactions in cancer cells| Molecular systems biology |2018|research article Cover|
Caicedo J. C., Cooper S., **Heigwer F.** et al. | Data-analysis strategies for image-based cell profiling| Nature Methods |2017| Review, Cover|
Scheeder C.*, **Heigwer F.** * & Boutros M.| HTSvis: a web app for exploratory data analysis and visualization of arrayed high-throughput screens| Bioinformatics |2017|research article|
Rauscher B.*, **Heigwer F.** * et al.| GenomeCRISPR - a database for high-throughput CRISPR/Cas9 screens| Nucleic Acids Research |2017|research article|
**Heigwer F.***, Zhan T*. et al.| CRISPR library designer CLD: software for multispecies design of single guide RNA libraries| Genome Biology |2016|research article|
Boutros M., **Heigwer F.**, Laufer C.| Microscopy-Based High-Content Screening| Cell |2015| review|
Winter J., Breinig M., **Heigwer F.** et al.| caRpools: An R package for exploratory data analysis and documentation of pooled CRISPR/Cas9 screens| Bioinformatics |2015| research article|
Volz H. C.*, **Heigwer F.** * et al.| Single-cell phenotyping of human induced pluripotent stem cells by high-throughput imaging| bioRxiv |2015 | preprint |
**Heigwer, F.** et al.| E-CRISP: Fast CRISPR target site identification| Nature Methods |2014|research article|
**Heigwer, F.** et al.| E-TALEN: a web tool to design TALENs for genome engineering| Nucleic acids research |2013|research article|
Kitanovic, A. *, Bonowski, F. *, **Heigwer, F.** * et al.| Acetic acid treatment in S. cerevisiae creates significant energy deficiency and nutrient starvation that is dependent on the activity of the mitochondrial transcriptional complex Hap2-3-4-5| Front. Oncol. |2012|research article|
Holenya, P. *, **Heigwer, F.** * and Wölfl, S.| KOMA: ELISA-microarray calibration and data analysis based on kinetic signal amplification| Journal of Immunological Methods |2012|research article|
Holenya, P. *, Kitanovic, I. *, **Heigwer, F.** and Wölfl, S.| Microarray-based kinetic colorimetric detection for quantitative multiplex protein phosphorylation analysis| Proteomics |2011| research article|

<span style="color:gray;font-size: .5rem">_* contributed equally_</span>
